content/syllabuses/data-eng-part-1.md

```diff
@@ -6,6 +6,8 @@ _db_id: 4
 - {{% contentlink path="topics/solo-learn/python/module-5-more-types" %}}
 - {{% contentlink path="agile/agile-scrum" %}}
 - {{% contentlink path="topics/clean-code/python" %}}
+- {{% contentlink path="topics/file-and-directory-naming/general" %}}
```

Comment:
since the `general` topic is a prerequisite to the python topic, we dont need to mention both of them. Just the python one.

Tilde will make sure it adds all the prerequisites to people's boards.